code/game/objects/items/weapons/implants/implantchair.dm

```diff
@@ -74,15 +74,16 @@
 			return
 
 
-	attackby(var/obj/item/G as obj, var/mob/user as mob)
-		if(istype(G, /obj/item/grab))
-			if(!ismob(G:affecting))
+	attackby(var/obj/item/grab, var/mob/user as mob)
+		if(istype(grab, /obj/item/grab))
+			var/obj/item/grab/G = grab
+			if(!ismob(G.affecting))
 				return
-			for(var/mob/living/carbon/slime/M in range(1,G:affecting))
-				if(M.Victim == G:affecting)
-					to_chat(usr, "[G:affecting:name] will not fit into the [src.name] because they have a slime latched onto their head.")
+			for(var/mob/living/carbon/slime/M in range(1,G.affecting))
+				if(M.victim == G.affecting)
+					to_chat(usr, span("warning", "[G.affecting.name] will not fit into the [src.name] because they have a slime latched onto their head."))
```

Comment:
```suggestion
					to_chat(usr, span("warning", "[G.affecting.name] will not fit into \the [src.name] because they have a slime latched onto their head."))
```